react/features/settings/components/AbstractSettingsView.js

```diff
@@ -168,13 +176,15 @@ export class AbstractSettingsView<P: Props, S: *> extends Component<P, S> {
  * @param {Object} state - The redux state.
  * @protected
  * @returns {{
+ *     _serverURLChangeAllowed: boolean,
  *     _serverURL: string,
  *     _settings: Object,
  *     _visible: boolean
  * }}
  */
 export function _mapStateToProps(state: Object) {
     return {
+        _serverURLChangeAllowed: isServerURLChangeAllowed(state),
```

Comment:
Move this to the `_mapStateToProps` implementation in native/SettingsView.js, since it has no web relevance.